internal/output/table.go

```diff
@@ -100,6 +104,36 @@ func tableBuilderInner(vulnResult *models.VulnerabilityResults, addStyling bool,
 
 				outputRow = append(outputRow, strings.Join(links, "\n"))
 
+				var outputSeverities []string
+				for _, vulnID := range group.IDs {
+					var severities []models.Severity
+					for _, vuln := range pkg.Vulnerabilities {
+						if vuln.ID == vulnID {
+							severities = vuln.Severity
+						}
+					}
+					for i, severity := range severities {
+						if i != 0 {
+							outputSeverities = append(outputSeverities, ", ")
+						}
+
```

Comment:
```suggestion
```
I don't think this is correct, it'll append a new line to just a comma when you do a join on line 135. I think you just need to add `,`  to line 135

Reply:
Right, thanks for the suggestion! Changed it in the latest commit.